File: backend/src/main/java/com/wgzhao/addax/admin/repository/SysDictRepo.java
package com.wgzhao.addax.admin.repository;

import com.wgzhao.addax.admin.model.SysDict;
import org.springframework.data.jpa.repository.JpaRepository;

public interface SysDictRepo
    extends JpaRepository<SysDict, Integer>
{

    SysDict findByCode(int code);
}

File: src/main/kotlin/com/wgzhao/addax/admin/repository/SysDictRepo.kt
package com.wgzhao.addax.admin.repository

import com.wgzhao.addax.admin.model.SysDict
import org.springframework.data.jpa.repository.JpaRepository

interface SysDictRepo

    : JpaRepository<SysDict?, Int?> {
    fun findByCode(code: Int): SysDict?
}
